kolibri/plugins/device_management/assets/src/views/ManageContentPage/SelectNetworkAddressModal/AddAddressForm.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+
+  <KModal
+    :title="$tr('header')"
+    :submitText="$tr('submitButtonLabel')"
+    :cancelText="$tr('cancelButtonLabel')"
+    size="medium"
+    @submit="handleSubmit"
+    @cancel="$emit('cancel')"
```

Comment:
Here, the cancel button goes back, whereas in the Select network address modal, the cancel button cancels, rather than going back to the Import modal. A bit inconsistent.

Reply:
@khangmach @jtamiace